src/adapters/logger.ts

```diff
@@ -1,7 +1,11 @@
 import pino from 'pino'
 
 export const logger = pino({
-  transport: {
-    target: 'pino-pretty'
-  }
+  ...(process.env.LOGGER_TRANSPORT
+    ? {
+        transport: {
+          target: process.env.LOGGER_TRANSPORT
```

Comment:
would be nice to have a default transport to `pino-pretty` if LOGGER_TRANSPORT is not defined